src/components/list-illustration/list-illustration.scss

```diff
@@ -290,3 +298,146 @@ $list-illustration: null !default;
     }
   }
 }
+
+.ecl-list-illustration__divider {
+  padding: 0;
+  margin: 0;
+
+  .ecl-separator {
+    max-width: map.get($list-illustration, 'divider-max-width');
+    padding-block-end: var(--s-s);
+  }
+
+  @include breakpoints.up('m') {
```

Comment:
Open question here: shouldn't we use container queries instead? List with illustration could be placed in smaller container

Reply:
So, i didn't because in the only example available in figma with the component placed in a layout with columns, a media query is used, i agree that we are progressively switching, but let's do it consistently in all the component here, in case.